modules/javafx.graphics/src/main/native-glass/win/GlassWindow.cpp

```diff
@@ -510,6 +510,9 @@ LRESULT GlassWindow::WindowProc(UINT msg, WPARAM wParam, LPARAM lParam)
         case WM_RBUTTONDBLCLK:
         case WM_MBUTTONUP:
         case WM_MBUTTONDBLCLK:
+        case WM_XBUTTONDOWN:
```

Comment:
That's a really good question and good catch. I think that it should indeed to moved so that the window will grab focus on BACK/FORWARD mouse button clicks. I'll make that change now.